scripts/src/main/resources/scripts/migration

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/functions
+
+from_version=2023.01.001
+if [ -f "${DEVON_IDE_HOME}/.devon.software.version" ]  
+then
+  devon_version="${DEVON_IDE_HOME}/.devon.software.version"
```

Comment:
should be defined outside the if (and used for if condition). Otherwise line 31 can fail.

Reply:
Done. Thanks.